websites/H/HentaiVN/presence.ts

```diff
@@ -0,0 +1,570 @@
+			.querySelectorAll("span[itemprop='name']")[2]
+			.textContent.trim()}"`;
+		presenceData.state = document
+			.querySelectorAll("span[itemprop='name']")[3]
+			.textContent.trim();
+		presenceData.smallImageKey = "doc";
+		presenceData.largeImageKey = "truyen";
+		presenceData.buttons = [
+			{
+				label: "Đọc cùng",
+				url: document.URL
+			}
+		];
+	} else if (
+		!document.location.pathname.startsWith("/forum/t") &&
+		document.location.pathname.includes("-doc-truyen-")
+	) {
+		presenceData.details = `Đang xem thông tin truyện "${document
+			.querySelectorAll("span[itemprop='name']")[2]
+			.textContent.trim()}"`;
+		presenceData.buttons = [
+			{
+				label: "Xem truyện",
+				url: document.URL
+			}
+		];
+		presenceData.smallImageKey = "xem";
+		presenceData.largeImageKey = "truyen";
+	} else if (document.location.pathname.startsWith("/forum/t")) {
+		presenceData.details = `Đang xem topic "${document
+			.querySelectorAll("span[itemprop='name']")[3]
+			.textContent.trim()}"`;
+		presenceData.buttons = [
+			{
+				label: "Xem topic",
+				url: document.URL
+			}
+		];
+		presenceData.smallImageKey = "xem";
+		presenceData.largeImageKey = "forum";
+	} else if (document.location.pathname.startsWith("/g/")) {
+		if (document.location.search.includes("page")) page = inURL.get("page");
+
+		presenceData.details = `Đang xem danh sách truyện của nhóm "${document
+			.querySelectorAll("span[itemprop='name']")[2]
+			.textContent.trim()}"`;
+		presenceData.state = `Trang ${page}`;
+		presenceData.smallImageKey = "list";
+		presenceData.largeImageKey = "truyen";
+	} else if (document.location.pathname === "/forum/") {
+		presenceData.details = "Đang duyệt forum";
+		presenceData.smallImageKey = "surf";
+	} else if (document.location.pathname.startsWith("/forum/f")) {
+		presenceData.details = `Đang duyệt box ${document
+			.querySelector("b")
+			.textContent.trim()}`;
+		presenceData.smallImageKey = "surf";
+		presenceData.largeImageKey = "forum";
+	} else if (document.location.pathname.startsWith("/forum/c")) {
+		largebox = document.querySelector("title").textContent.trim();
+		largebox = largebox.substring(0, largebox.indexOf("-"));
+		presenceData.details = `Đang duyệt danh mục ${largebox}`;
+		presenceData.smallImageKey = "surf";
+		presenceData.largeImageKey = "forum";
+	} else if (document.location.pathname.startsWith("/forum/search-plus.php")) {
+		presenceData.details = "Đang tìm kiếm truyện nâng cao";
+		presenceData.smallImageKey = "search";
+		presenceData.largeImageKey = "home";
+		if (document.location.search.includes("search")) {
+			keystr = inURL.getAll("tag[]");
+			namekey = `Từ khóa: ${inURL.get("name")} - `;
+			doukey = `Doujin: ${inURL.get("dou")} - `;
+			charkey = `Nhân vật: ${inURL.get("char")} - `;
+			if (namekey.length === 12) namekey = "";
+
+			if (doukey.length === 11) doukey = "";
+
+			if (charkey.length === 13) charkey = "";
+
+			if (keystr.length === 0) keystr = "";
```

Comment:
Use ternaries instead

Reply:
Can't use ternaries to replace because there is a variable  not sure will appear in the middle of ternaries